streams_chunks_test.go

```diff
@@ -83,24 +67,16 @@ func TestChunkUtf8String(t *testing.T) {
 
 		chunks := chunkUtf8String(testString)
 
-		if len(chunks) != 2 {
-			t.Errorf("Expected 2 chunks, got %d", len(chunks))
-		}
+		require.Len(t, chunks, 2)
```

Comment:
quick note on use on `require`. The convention is to have the `expected` value as the second argument and the `actual` value as the third argument. Can you please have a look and ensure that convention is followed. Sorry for nit-picking, but just want to keep it consistent (I am probably guilty of not following this in several places, but I am correcting things as and when I see it :-). )

Reply:
You're right, that is the convention for `require.Equal()`, which is used correctly, I've double-checked.

- `require.Zero()` and `require.True()` only take 1 value that they match with zero/true, so that is correct as well.
- That leaves us with `require.Len()` takes an object type that supports `len()` as second parameter and the expected length as the third: https://pkg.go.dev/github.com/stretchr/testify/require#Len

Reply:
Oh cool, did not know that Len has a different convention. Thank you 🙏